tailwind.config.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
       },
       height: {
         'half-screen': 'calc(50vh - 3rem)',
-        'full-screen': 'calc(100vh - 6rem)',
+        'full-screen': 'calc(100vh - 3rem)',
```

Comment:
3 rem from the header, 3 rem from the part of the footer that sticks up from under the page.